worldengine/generation.py

```diff
@@ -105,12 +105,20 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
     """
     e = world.elevation['data']
     ocean = fill_ocean(e, ocean_level)
-    hl = find_threshold_f(e, 0.10)  # the highest 10% of all (!) land are declared hills
-    ml = find_threshold_f(e, 0.03)  # the highest 3% are declared mountains
+
+    # values chosen to emulate the exponential behavior seen here:
+    # http://www.ngdc.noaa.gov/mgg/global/etopo1_surface_histogram.html
+    pl = find_threshold_f(e, 0.95, ocean=ocean)  # the highest x% of land are declared plains; save some for beaches (?)
+    hl = find_threshold_f(e, 0.50, ocean=ocean)  # the highest x% are declared hills
```

Comment:
Actually my former comment in the code was off: *Plains* used to be the top 10% of the world, hills only the top 3%. The current value pushes that to 15%. So if we wanted less, we should pick a base other than 2...so more like 10^x? That sounds very drastic to me.

Don't forget, though, that hills were the highest portion of the world before the new levels were added. From that point of view it might be perfectly fine to have 50% hills and maybe equate the new medium mountains with the former hills.